lib/utils/logging.js

```diff
@@ -102,36 +116,40 @@ export function writeErrorLogFile (destination, errorLog) {
     })
 }
 
-export function setupErrorLogging (errorLog) {
+// Init listeners for log messages, transform them into proper format and logs/displays them
+export function setupLogging (log) {
   function errorLogger (level, error) {
-    errorLog.push({
+    const logMessage = {
       ts: (new Date()).toJSON(),
+      level,
       [level]: error
-    })
+    }
+    if (level !== 'info') {
+      log.push(logMessage)
+    }
+    logEmitter.emit('display', logMessage)
   }
 
   logEmitter.addListener('warning', (error) => errorLogger('warning', error))
   logEmitter.addListener('error', (error) => errorLogger('error', error))
+  logEmitter.addListener('info', (error) => errorLogger('info', error))
 }
 
+// Format log message to display them as task status
 export function logToTaskOutput (task) {
```

Comment:
adding the `display` event, the formatting of task output is way easier. That's the live task status the users sees while importing.